src/core/xref.js

```diff
@@ -11,6 +11,11 @@ import {
   showInlineWarning,
 } from "./utils";
 import { openDB } from "idb";
+import { pub } from "./pubsubhub";
+
+const profiles = {
+  w3c: ["HTML", "INFRA", "URL", "PROMISES-GUIDE", "WEBIDL", "DOM", "FETCH"],
```

Comment:
Promise Guide is being kinda crappy... and I think it's being folded into the WebIDL spec... so let's not include it. 
```suggestion
  w3c: ["HTML", "INFRA", "URL", "WEBIDL", "DOM", "FETCH"],
```